packages/common/lib/common/tools.js

```diff
@@ -2450,51 +2503,49 @@ function validateGeneralObjectProperties(obj, extend) {
  * @alias getAllInstances
  * @memberof tools
  * @param {string[]} adapters list of adapter names to get instances for
+ * @param {object} objects class redis objects
  * @param {function} callback callback to be executed
  */
 function getAllInstances(adapters, objects, callback) {
+    showDeprecatedMessage('tools.getAllInstances');
```

Comment:
Do we really need this, is someone from outside accessing js-controller tools, else we should just fix all occurences.

Reply:
It is for me. Later when I could see no one uses getAllInstances, I will remove it all. It is hard to change ALL functions on ALL places at once.